IntegerManipulationPractise/Shuffle.cs

```diff
@@ -10,13 +10,35 @@ namespace IntegerManipulationPractise
     class Shuffle
     {
         //Take list of numbers, shuffle it, return it...
-        public static List<int> ShuffleList(List<int> list)
+        public static List<int> ShuffleListRandomInsert(List<int> list)
         {
+            var recivedList = list;
             var retVal = new List<int>();
+            var random = new Random();
+            foreach (var element in recivedList)
+            {
+                retVal.Insert(random.Next(0, retVal.Count+1), element);//mora +1 zato sto nikada nece prebaciti prvi broj...
+            }
+            return retVal;
+        }
```

Comment:
nisi nista propustio, za sada sam samo izgenerisao listu, odradio shuffle posto deljenje na 10 manjih lista ne verujem da ce mi biti problem, pa sam poceo od ovoga, pa ako ti se svidja, da odradim deljenje na liste... btw, sada cu da odradim deljenje na manje liste, error ako ne upisu broj(DAMN, to moram da se potsetim, to sam radio u BJack-u, ali sam zaboravio potpuno) ne znam da li si dobio ove komentare, ali ovo prvi put radim(pull request)